ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -158,7 +158,8 @@ DEFAULT_PROTOCOL = "https"
 
 ANALYTICS_IDS = {
     'GOOGLE_ANALYTICS_ID': '{{ localsettings.GOOGLE_ANALYTICS_ID }}',
-    'PINGDOM_ID': '{{ localsettings.PINGDOM_ID }}'
+    'PINGDOM_ID': '{{ localsettings.PINGDOM_ID }}',
+    'SEGMENT_ANALYTICS_KEY': '{{ localsettings.SEGMENT_ANALYTICS_KEY }}',
```

Comment:
fyi @sravfeyn @kaapstorm if you add an environment variable like this you need to make sure it exists in all environments (this was missing from india) or ansible will choke on it. you can do a more protective style like https://github.com/dimagi/commcarehq-ansible/pull/70/ if you don't want to add it everywhere